skyvern/core/script_generations/skyvern_page.py

```
@@ -750,15 +906,60 @@ async def select_option(
         else:
-            locator = self.page.locator(f"xpath={xpath}")
+            locator = self.page.locator(selector)
             await locator.select_option(option_value, timeout=timeout)
         return option_value
 
+    @action_wrap(ActionType.SELECT_OPTION)
+    async def select_option(
+        self,
+        selector: str,
+        value: str | None = None,
+        label: str | None = None,
```

Ruff (0.14.0)

Unused method argument: `label`

(ARG002)